docs/openapi/api-external-b2b-pa-bundle.yaml

```diff
@@ -907,7 +913,7 @@ components:
           type: string
           description: Tipologia comunicazione fisica
           enum:
-            - SIMPLE_REGISTERED_LETTER
+            - AR_REGISTERED_LETTER
```

Comment:
questa modifica anche se sulle API bundle non deve ancora essere portata in main